src/main/java/org/polypheny/jdbc/properties/DriverProperties.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019-2024 The Polypheny Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.polypheny.jdbc.properties;
+
+import java.util.TimeZone;
+import lombok.Getter;
+
+public class DriverProperties {
+
+    @Getter
+    private static final String DRIVER_NAME = "JDBC driver for PolyphenyDB";
+    @Getter
+    //TODO TH can we automate this?
+    private static final int DRIVER_MAJOR_VERSION = 2;
+    @Getter
+    //TODO TH can we automate this?
+    private static final int DRIVER_MINOR_VERSION = 0;
+    @Getter
+    //TODO TH can we automate this?
+    private static final String DRIVER_VERSION_QUALIFIER = "-SNAPSHOT";
+    @Getter
+    private static final String DRIVER_VERSION = DRIVER_MAJOR_VERSION + '.' + DRIVER_MINOR_VERSION + DRIVER_VERSION_QUALIFIER;
+    @Getter
+    private static final boolean JDBC_COMPLIANT = false;
```

Comment:
Why is this false?

Reply:
This was set when starting work on the new driver and had not been updated since. The current version is compliant with JDBC 4.2.

Reply:
Perfect